LBH.AdultSocialCare.Api/V1/Controllers/StatusController.cs

```diff
@@ -64,6 +80,12 @@ public async Task<ActionResult<StatusResponse>> Get(Guid statusId)
             }
         }
 
+        /// <summary>Gets all.</summary>
+        /// <returns>The List of Status model</returns>
+        [ProducesResponseType(typeof(IList<Status>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status404NotFound)]
```

Comment:
Get all can return empty array instead of not found

Reply:
Thanks!